Modules/IO/GDCM/include/itkGDCMImageIO.h

```diff
@@ -164,6 +164,32 @@ class ITKIOGDCM_EXPORT GDCMImageIO : public ImageIOBase
   itkGetConstMacro(LoadPrivateTags, bool);
   itkBooleanMacro(LoadPrivateTags);
 
+  /** Convert input Y'CbCr to RGB, enabled by default.
+   */
+  itkSetMacro(InputYBRtoRGB, bool);
+  itkGetConstMacro(InputYBRtoRGB, bool);
+  itkBooleanMacro(InputYBRtoRGB);
```

Comment:
It be clearer to me if "Input" was replaced with "Read" and if needed "Output"->"Write".

Reply:
@blowekamp  Do you mean ReadYBRtoRGB and WriteRGBtoYBR?I'll change.